tests/PHPUnit/Integration/DataAccess/ArchiveSelectorTest.php

```diff
@@ -172,7 +172,7 @@ public function getTestDataForGetArchiveIdAndVisits()
                 '',
                 $minDateProcessed,
                 false,
-                [false, false, false, true],
+                [false, false, false, false],
```

Comment:
@diosmosis I suppose this is why we create now more archives below in eg https://github.com/matomo-org/matomo/pull/15887/files#diff-dcb3557abe94edd8776f637afdfb76b7R99

Do you know if this was a bug before that we would have assumed there was an archive when there was no done flag? Trying to understand how this used to work. Maybe in the past at the beginning of Matomo there were no done flags? No idea...

Reply:
I think before we would use any number of visits even if it wasn't the latest archive. So if there was:

- an archive w/ ts_archived = 14:23, no nb_visits
- an archive w/ ts_archived = 02:55, nb_visits = 5

We'd say there were 5 visits. Which means if in between if someone deleted all the visits it would still think there are 5. so probably a bug.

Reply:
👍